impl/internal/did/did.go

```diff
@@ -16,13 +17,22 @@ import (
 )
 
 type (
-	DHT string
+	DHT       string
+	TypeIndex int
 )
 
 const (
 	// Prefix did:dht prefix
 	Prefix               = "did:dht"
 	DHTMethod did.Method = "dht"
+
+	Organization           TypeIndex = 1
+	GovernmentOrganization TypeIndex = 2
+	Corporation            TypeIndex = 3
+	LocalBusiness          TypeIndex = 4
+	SoftwarePackage        TypeIndex = 5
+	WebApplication         TypeIndex = 6
+	FinancialInstitution   TypeIndex = 7
```

Comment:
@michaelneale this is how we'll be encoding types, alongside this https://github.com/TBD54566975/did-dht-method/pull/29/files#diff-c4795c497b83a8c03e33535caf0fb0e1512cecd8cb448f62467326277c152afeR336